Source/Blazorise/Components/Bar/Bar.razor.cs

```diff
@@ -72,6 +72,18 @@ protected override async Task OnFirstAfterRenderAsync()
                     await Toggle();
                 }
             }
+            else
+            {
+                if ( string.IsNullOrEmpty( Theme?.BarOptions?.HorizontalHeight ) )
+                {
+                    var rect = await JSRunner.GetElementInfo( ElementRef, ElementId );
+
+                    if ( rect.BoundingClientRect.Height > 0 )
+                    {
+                        await JSRunner.AddThemeVariable( ThemeVariables.HorizontalBarHeight, $"{rect.BoundingClientRect.Height}px" );
```

Comment:
Enough to be on firstRender? Does it ever need any recalc, if for some reason the users are able to change?

Reply:
I was thinking about it. But then again how many times do you think topbar height will be changed. I think very rarely. In case it does get reported I can just move it to recalculate in OnAfterRender.